src/api/routes/summaries.ts

```diff
@@ -180,3 +190,48 @@ export async function findMissingCrudeSummaries(): Promise<CrudeSummaryDescripto
   }
   return ret;
 }
+
+export async function extractHandlebars(transcriptId: string): Promise<string[]> {
```

Comment:
if you always call extrcthandlebars and createNameMap together, either create a wrapper function so callers only call this single wrapper function (and do not export the other two functions), or merge these two functions into one.